activesupport/README.rdoc

```diff
@@ -5,6 +5,7 @@ extensions that were found useful for the Rails framework. These additions
 reside in this package so they can be loaded as needed in Ruby projects
 outside of Rails.
 
+You can read more about Active Support in the {Active Support Core Extensions}[https://edgeguides.rubyonrails.org/active_support_core_extensions.html] guide.
```

Comment:
This isn't like the other guides. Let's fit in the wording with the above paragraph ala "You can read more about the extensions in the {Active Support Core Extensions} guide."

Reply:
Sure, I am doing it right away. :)

Reply:
@kaspth It's done :)